### PR TITLE
test passing for reverse.py

### PR DESCRIPTION
**Pull request by noahfranco**

### CODE REVIEW

**Comment by noahfranco** [comment]
@mjherich met MVP :)

File: names/binary_search_tree.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            if not self.right:
+                self.right = BinarySearchTree(i) # we assign to the right part of the binary search tree
+            else:
+                self.right.insert(i) # otherwise we will add (i == our value) to the right of the binary search tree
+        
+
+
+    # Return True if the tree contains the value
+    # False if it does not
+    def contains(self, target):
```
Comment (mjherich) [inline names/binary_search_tree.py:31]: Good job fixing the contains function!

File: ring_buffer/ring_buffer.py [hunk 2]
```diff
@@ -7,14 +7,43 @@ def __init__(self, capacity):
         self.current = None
         self.storage = DoublyLinkedList()
 
-    def append(self, item):
-        pass
 
+# *** Plan ***
```
Comment (mjherich) [inline ring_buffer/ring_buffer.py:11]: Thanks for commenting your plan here. It communicates that you are building a good problem solving framework!

**Review by mjherich — APPROVED** [review]
Great job 🔥 🔥 🔥

**Pull request merged**